src/main/java/id/co/veritrans/sdk/core/VeritransRestAdapter.java

```diff
@@ -24,118 +22,94 @@
+                    .setConverter(new GsonConverter(gson))
+                    .setLogLevel(RestAdapter.LogLevel.FULL)
+                    .setClient(new OkClient(okHttpClient));
+            RestAdapter restAdapter;
+            if (BuildConfig.DEBUG) {
+                restAdapter = builder.build();
+                restAdapter.setLogLevel(RestAdapter.LogLevel.FULL);
+            } else {
+                restAdapter = builder.build();
             }
-            return paymentAPI;
+            return restAdapter.create(PaymentAPI.class);
         } else {
-            if (showNetworkNotAvailableDialog) {
-                VeritransBusProvider.getInstance().post(new NetworkUnavailableEvent());
-            }
+            return null;
         }
-        return null;
     }
 
-    public static PaymentAPI getMerchantApiClient(boolean showNetworkNotAvailableDialog) {
+    public static PaymentAPI getMerchantApiClient() {
         VeritransSDK veritransSDK = VeritransSDK.getVeritransSDK();
         if (veritransSDK != null
                 && veritransSDK.getContext() != null
                 && Utils.isNetworkAvailable(veritransSDK.getContext())) {
 
-            if (merchantPaymentAPI == null && VeritransSDK.getVeritransSDK() != null) {
-
-                OkHttpClient okHttpClient = new OkHttpClient();
-                okHttpClient.setConnectTimeout(60, TimeUnit.SECONDS);
-                Gson gson = new GsonBuilder()
-                        .setFieldNamingPolicy(FieldNamingPolicy.IDENTITY)
-                        .registerTypeAdapter(Date.class, new DateTypeAdapter())
-                        .create();
-                RestAdapter.Builder builder = new RestAdapter.Builder()
-                        .setConverter(new GsonConverter(gson))
-                        .setLogLevel(RestAdapter.LogLevel.FULL)
-                        .setClient(new OkClient(okHttpClient));
-                RestAdapter restAdapter;
-
-                builder.setEndpoint(VeritransSDK.getVeritransSDK().getMerchantServerUrl());
-                restAdapter = builder.build();
-
-                if (BuildConfig.DEBUG) {
-                    restAdapter.setLogLevel(RestAdapter.LogLevel.FULL);
-                }
-
-                merchantPaymentAPI = restAdapter.create(PaymentAPI.class);
-
-            } else if (VeritransSDK.getVeritransSDK() == null) {
-                Logger.e(Constants.ERROR_SDK_IS_NOT_INITIALIZED);
+            OkHttpClient okHttpClient = new OkHttpClient();
+            okHttpClient.setConnectTimeout(60, TimeUnit.SECONDS);
+            Gson gson = new GsonBuilder()
+                    .setFieldNamingPolicy(FieldNamingPolicy.IDENTITY)
+                    .registerTypeAdapter(Date.class, new DateTypeAdapter())
+                    .create();
+            RestAdapter.Builder builder = new RestAdapter.Builder()
+                    .setConverter(new GsonConverter(gson))
+                    .setLogLevel(RestAdapter.LogLevel.FULL)
```

Comment:
I think this should be removed, otherwise on release this will still produce log